CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased changes
 
+## concordium-node 1.1.0
+
 - Update dependencies, code cleanup, and removal of the staging_net feature. No functional changes.
 - Fix a bug in average throughput calculation that was triggered in some cases
   of bad connectivity.
```

Comment:
Perhaps we should credit kryptomouse for this?

Reply:
We have not named individual contributions like this for other changes, so it would not be correct to single out this instance.

I also don't think this belongs to the changelog. It would make sense in release notes to bring attention to external contributions.